share/spice/hacker_news/hacker_news.js

```diff
@@ -12,80 +12,55 @@
             return Spice.failed('hacker_news');
         }
 
-        Spice.add({
-            id: 'hacker_news',
-            name: 'Social',
-            data: api_result.hits,
-            meta: {
-                sourceName: 'HN Search',
-                sourceUrl: sourceUrl,
-                total: api_result.hits,
-                itemType: (api_result.hits.length === 1) ? 'Hacker News submission' : 'Hacker News submissions',
-                searchTerm: decodeURIComponent(query)
-            },
-            normalize: function(item) {
-                return {
-                    title: item.title,
-                    url: (item.url) ? item.url : 'https://news.ycombinator.com/item?id=' + item.objectID,
-                    points: item.points || 0,
-                    num_comments: item.num_comments || 0,
-                    post_domain: extractDomain(item.url),
-                    date_from: fuzzyDate(now - item.created_at_i),
-                    arrowUrl: DDG.get_asset_path('hacker_news','arrow_up.png'),
-                    id: item.objectID
-                };
-            },
-            templates: {
-                group: 'text',
-                options: {
-                    footer: Spice.hacker_news.footer
+        DDG.require('moment.js', function(){
+            Spice.add({
+                id: 'hacker_news',
+                name: 'Social',
+                data: api_result.hits,
+                meta: {
+                    sourceName: 'HN Search',
+                    sourceUrl: sourceUrl,
+                    total: api_result.hits,
+                    itemType: (api_result.hits.length === 1) ? 'Hacker News submission' : 'Hacker News submissions',
+                    searchTerm: decodeURIComponent(query)
                 },
-                detail: false,
-                item_detail: false,
-                variants: {
-                    tileTitle: "3line-small",
-                    tileFooter: "3line"
-                }
-            },
-            sort_fields: {
-                score: function(a, b){
-                    return (a.points > b.points) ? -1 : 1;
+                normalize: function(item) {
+                    return {
+                        title: item.title,
+                        url: (item.url) ? item.url : 'https://news.ycombinator.com/item?id=' + item.objectID,
+                        points: item.points || 0,
+                        num_comments: item.num_comments || 0,
+                        post_domain: extractDomain(item.url),
+                        date_from: moment(item.created_at_i * 1000).fromNow(),
+                        arrowUrl: DDG.get_asset_path('hacker_news','arrow_up.png'),
+                        id: item.objectID
```

Comment:
Actually, it seems like referencing `objectID` in the template is grabbing some other value...the links to the comments have much larger numbers with dashes in them and don't work for me...